tests/integration/release/parallel_test_launcher.py

```diff
@@ -0,0 +1,218 @@
+import csv
+import glob
+import os
+import shutil
+import subprocess
+import time
+from collections.abc import Sequence
+from datetime import datetime
+from pathlib import Path
+
+import pandas as pd
+
+
+def create_slurm_script(row: dict[str, str], script_name: str, output_dir: str) -> None:
+    dataset = row["dataset"]
+    engine = row["engine"]
+    memory = row["memory"]
+    time_limit = row["time"]  # DD:HH:MM
+    partition = (
+        "all.q" if int(time_limit.split(":")[0]) <= 24 else "long.q"
+    )  # long.q if 24 longer than 24 hours
+    release_tests_dir = Path(__file__).parent
+    pytest_command = (
+        f"pytest -rA --release --dataset {dataset} --engine {engine} {release_tests_dir}"
```

Comment:
`-A`? What is that?

Reply:
It shows the pytest output in a more readable way.

Reply:
Weird, it doesn't show up in `pytest --help` 🤷‍♂️